internal/notification/notification.go

```diff
@@ -0,0 +1,35 @@
+package notification
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"gocloud.dev/pubsub"
+)
+
+type pkgIdentifier struct {
+	Name      string
+	Version   string
+	Ecosystem string
+}
+
+type NotificationMessage struct {
+	Package	pkgIdentifier
+}
+
+func PublishNotification(ctx context.Context, notificationTopic *pubsub.Topic, name, version, ecosystem string) error {
```

Comment:
This name could be more descriptive about what kind of notification is being sent.

Reply:
+1. Also, we can avoid stutter by not having "Notification" in the name.

Something like "PublishAnalysisCompletion"  ?

Reply:
Also same comment for the struct type itself -- we can name it "AnalysisCompletion"